lib/hyrax/specs/shared_specs/indexers.rb

```diff
@@ -61,9 +82,37 @@
   end
 end
 
+RSpec.shared_examples 'a Core metadata indexer' do
```

Comment:
This is mostly the same with the similar changes for variable validation and removing the default resource.  It was moved to make the ordering of the various shared specs easier to follow.